### ISSUE TITLE
sbt-release 1.0.14 doesn't exist

### ISSUE THREAD

**Issue by larsrh** [sbt/sbt-release#279]
scala-steward created lots of PRs for sbt-release 1.0.14, but it doesn't exist on Bintray

![Screenshot_2021-02-21 Build software better, together](https://user-images.githubusercontent.com/923079/108626116-cf5a3a00-744e-11eb-8a56-d84b366eb917.png)

```
[error] sbt.librarymanagement.ResolveException: Error downloading com.github.sbt:sbt-release;sbtVersion=1.0;scalaVersion=2.12:1.0.14
[error]   Not found
[error]   Not found
[error]   not found: repo1.maven.org/maven2/com/github/sbt/sbt-release_2.12_1.0/1.0.14/sbt-release-1.0.14.pom
[error]   not found: /home/runner/.ivy2/localcom.github.sbt/sbt-release/scala_2.12/sbt_1.0/1.0.14/ivys/ivy.xml
[error]   not found: repo.scala-sbt.org/scalasbt/sbt-plugin-releases/com.github.sbt/sbt-release/scala_2.12/sbt_1.0/1.0.14/ivys/ivy.xml
[error]   not found: repo.typesafe.com/typesafe/ivy-releases/com.github.sbt/sbt-release/scala_2.12/sbt_1.0/1.0.14/ivys/ivy.xml

```

**Comment by xuwei-k** [comment]
Sorry, I have published 1.0.15 🙇 
https://github.com/scala-steward-org/scala-steward/pull/1952
https://repo1.maven.org/maven2/com/github/sbt/sbt-release_2.12_1.0/

**Comment by larsrh** [comment]
Thanks!
